shimmer/src/commonMain/kotlin/com/valentinilk/shimmer/ShimmerBounds.kt

```diff
@@ -8,14 +8,21 @@ import androidx.compose.ui.geometry.Rect
 internal fun rememberShimmerBounds(
     shimmerBounds: ShimmerBounds,
 ): Rect? {
+    val screenInfo = shimmerBounds.rememberScreenInfoOrNull()
     return remember(shimmerBounds) {
```

Comment:
That is a bit of a tricky one. When the `ScreenInfo` is being updated, we need to update the `Rect` we are returning as well. So we'd at least need a `remember(shimmerBounds, screenInfo)` here.

But since the `ScreenInfo` and a `Rect` are almost the same, we could just do something like:
```
return when(shimmerBounds) {
    ShimmerBounds.Window -> rememberWindowBounds // This would be the `internal expect fun` instead of `rememberScreenInfo``
    ShimmerBounds.Custom -> Rect.Zero
    ShimmerBounds.View -> null
}
```

But there is another change within `Shimmer.kt` required. But I'll add it as soon as your PR is merged.